src/pages/data.jsx

```diff
@@ -5,6 +5,11 @@ import Web3 from "web3";
 
 const tacoAddr = "0x347cc7ede7e5517bd47d20620b2cf1b406edcf07"
 export const ritual_columns = [
+  {
+    header: "Id",
+    accessor: "id",
+    numeric: false,
```

Comment:
ID is an integer

Reply:
Addressed in [refactor: reorder ritual table columns](https://github.com/threshold-network/tacoscan/pull/7/commits/c77e9c21b8a418bcf7360cddf6cc507b6652703f)